README.md

```diff
@@ -14,7 +13,19 @@ Stable versions are available on the
 
 ## Publishing
 
-New releases are automatically published to WordPress.org via Travis CI (using [this script](bin/deploy.sh)) whenever the version in [hypothesis.php](hypothesis.php) is incremented and a new version is tagged. Translation files will be updated as well. This method is largely based on the process described by [Iain Poulson](https://github.com/polevaultweb) in [this blog post](https://deliciousbrains.com/deploying-wordpress-plugins-travis/), and also makes use of the [wp-cli](https://wp-cli.org) `i18n` command.
+Follow these steps to publish a new plugin version.
+
+1. **Update the package version** in `hypothesis.php`, `readme.txt` and `package.json`, and merge that change into the `main` branch[^1]. We use [Semantic Versioning](https://semver.org/#semantic-versioning-200).
+2. **Create a tag** pointing at the version-change commit and generate a **new GitHub release** (details follow). Publishing a GitHub release will kick off a GitHub Action that will publish the plugin to wordpress.org
+
+### Creating a GitHub release
+
+Create a [new GitHub release](https://github.com/hypothesis/frontend-shared/releases/new/) with these values:
+
+1.  _Tag_: Create a new tag for the release, targeting the `main` branch (your just-merged version bump should be at the tip)[^2]. The tag should match the version number, e.g. `v5.2.1`.
+2.  _Title_: Use the tag name.
+3.  Click the `Auto-generate release notes` button to generate release notes and edit as needed. We use [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) formatting.[^3]
+4.  Leave other fields alone/as defaults.
```

Comment:
I took this from frontend-shared. The only difference is that the version needs to be edited in `hypothesis.php` and `readme.txt`.